app/src/main/java/org/mozilla/focus/state/AppState.kt

```diff
@@ -16,12 +16,14 @@ import java.util.UUID
  * @property autoplayRulesChanged A flag which reflects the state of autoplay rules,
  * whether they have been updated or not
  * @property secretSettingsEnabled A flag which reflects the state of debug secret settings
+ * @property showEraseTabsCfr A flag which reflects the state of erase tabs CFR
  */
 data class AppState(
     val screen: Screen,
     val topSites: List<TopSite> = emptyList(),
     val autoplayRulesChanged: Boolean = false,
-    val secretSettingsEnabled: Boolean = false
+    val secretSettingsEnabled: Boolean = false,
+    val showEraseTabsCfr: Boolean = false
```

Comment:
nit: Food for thought. `AppState` seems to become a sink for all things. Maybe a `BrowserScreenState` would be nice to have in the future and it could contain this kind of properties (cfr that only appears in the browser fragment).

Reply:
`BrowserFragment` is already a complex class, to add now a `BrowserScreenState` implies too many changes which will be out of the scope. I will investigate more to find a better approach